lib/emoji.js

```diff
@@ -9,6 +9,11 @@ var emojiByName = require('./emoji.json');
  */
 var emojiNameRegex = /:([a-zA-Z0-9_\-\+]+):/g;
 
+/**
+ * regex to remove emoji key strings, e.g. "":coffee: coffee" -> coffee
+ */
+var removeNameRegex = /:.*?\:/;
```

Comment:
The `?` is unnecessary. `.*?` says:

* `.`: any character
* `*`: any number of times (including 0)
* `?`: optionally

I'm betting the regular expression shouldn't match `::`. So it can be simplified to:

```suggestion
var removeNameRegex = /:.+:/;
```